.github/workflows/php.yml

```diff
@@ -13,33 +13,35 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
-    name: Tests with PHP ${{ matrix.php-versions }}
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+          - "8.0"
+        dependencies:
+          - "locked"
```

Comment:
It looks like it will be difficult to get a `composer.lock` that works with all the PHP versions in the matrix. I recommend removing `composer.lock` and testing only lowest & highest combinations.

Reply:
Fixed